src/content/en/2020/markup.md

```diff
@@ -626,34 +704,42 @@ Using `target="_blank"` is known as a [security vulnerability](https://mathiasby
 <figcaption>{{ figure_link(caption="Blank relationships.", sheets_gid="1876528165", sql_file="pages_wpt_bodies_by_device.sql") }}</figcaption>
 </figure>
 
-As a rule of thumb—also [for usability reasons](https://www.nngroup.com/articles/new-browser-windows-and-tabs/)—, prefer not to use `target=_blank`. 
+As a rule of thumb and for [usability reasons](https://www.nngroup.com/articles/new-browser-windows-and-tabs/), prefer not to use `target="_blank"`. 
 
-<p class="note">Within the latest Safari and Firefox versions, setting <code>target="_blank"</code> on <code>a</code> elements implicitly provides the same <code>rel</code> behavior as setting <code>rel="noopener"</code>. This is currently being <a href="https://bugs.chromium.org/p/chromium/issues/detail?id=898942">implemented in Chromium</a> as well.</p>
+<p class="note">Within the latest Safari and Firefox versions, setting <code>target="_blank"</code> on <code>a</code> elements implicitly provides the same <code>rel</code> behavior as setting <code>rel="noopener"</code>. This is already <a href="https://chromium-review.googlesource.com/c/chromium/src/+/1630010">implemented in Chromium</a> as well and will land in Chrome 88.</p>
 
+{# TODO(editors, authors): The following two sections each read like conclusions. Consider merging. #}
 ## Status and trends
 
-We’ve sprinkled some observations throughout the chapter, and you’ll have made your own observations. At the end of this 2020 analysis, here are some things that stood out for us.
+We've sprinkled some observations throughout the chapter, and you'll have made your own observations. At the end of this 2020 analysis, here are some things that stood out for us.
```

Comment:
The duplicate “observations“ strikes me now. Replace the latter by “taken your own notes” or something like that?

Reply:
I'm struggling with this paragraph. It seems to say "We saw some things, you saw some things. Here's some things we saw."

I think we can simplify it for a start by cutting the middle clause since you cover it later with the final paragraph in this section:

> We're leaving this open to you. What are your observations? What has caught your eye? What do you think has taken a turn for the worse, and what has improved? Leave a comment to share your thoughts!

What about the following for the remaining pieces:

```suggestion
We've touched on some observations throughout the chapter, but as a reflection on the state of markup in 2020, here are some things that really stood out for us:
```

Reply:
I like it. I’d prefer “some more things that stood out” over “some things that really stood out”, as we haven’t made comments on these things any earlier.